plugins/synced_folders/rsync/command/rsync_auto.rb

```diff
@@ -48,6 +48,9 @@ def execute
           paths = {}
           ignores = []
           with_target_vms(argv) do |machine|
+            next if machine.state.id == :not_created
```

Comment:
FYI, this does not seem to be an issue with normal machines, as later on the sync_folders function ends up returning an empty hash so nothing is synced...but with docker machines since they both are technically the same "machine", it would attempt to sync docker containers that hadn't been created yet.